src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -305,6 +321,9 @@ public static class Builder {
     private long maxOutputSize = 0; // in bytes
     private Map<Context.Library, Set<String>> disabled = new HashMap<>();
 
+    private Set<String> restrictedMethods = new HashSet<>();
+    private Set<String> restrictedProperties = new HashSet<>();
```

Comment:
Would feel a bit safer if these were `ImmutableSet.of()`

Reply:
Thanks. I have changed it to `ImmutableSet`.